### DISCUSSION TITLE
How to check if kubeprism is running?

### DISCUSSION THREAD

**Discussion by evanrich** [siderolabs/talos#8113] [General]
This is a dumb question probably, but here goes.  How can i validate if kubeprism is working?   I've upgraded from 1.5.5 to 1.6.0, and per https://www.talos.dev/v1.6/kubernetes-guides/configuration/kubeprism/ it says it's enabled by default, but i have nothing in my machineconfigs.  I've tried nmaping port 7445 from another host and it says closed, but from the docs it only listens internally.  Is there some way to actually verify it's working before i point cilium at it instead of my external HAPROXY Loadbalancer?  Thanks!

**Comment by steverfrancis** [comment]
talosctl -n NODE get kubeprismstatus -o yaml
should do it.

**Comment by steverfrancis** [comment]
Oh - Talos doesn't apply new enabled defaults to existing systems. So if you create a new cluster with Talos 1.6 - it will have KubePrism. But upgrading an older cluster will not enable it.
You will have to apply a machine patch to do so.

**Comment by frezbo** [comment]
Kubeprism only listens on localhost

**Comment by evanrich** [comment]
Makes sense!  Thanks all, ive patched my current config.
